src/pages/CommonComponents/Footer.js

```diff
@@ -0,0 +1,56 @@
+import React, { Component } from 'react';
+import './Footer.scss';
+export default class Footer extends Component {
+  render() {
+    return (
+      <footer>
+        <section>
+          <span>
+            지금까지 <span className="footerCounter">얼마</span>의 평가
+          </span>
+        </section>
+        <section>
+          <div className="footerSiteDescription">
```

Comment:
scss nesting 하기 때문에 앞에 이렇게 BEM방법론처럼 footer 등의 prefix를 계속 붙여주실 필요는 없습니다!

```suggestion
          <div className="siteDescription">
```

Reply:
다슬님과 함께 확인했습니다 감사합니다~!!